.gitignore

```diff
@@ -3,3 +3,4 @@ bin
 obj
 *.user
 **/Properties/launchSettings.json
+/src/Draco.Compiler.Cli/test.draco
```

Comment:
Let's not litter the repo in the first place. We shouldn't need to ignore this, just delete on your side or put it in the binaries folder (or even outside the repo)